src/index.ts

```diff
@@ -59,8 +63,8 @@ const hasValidHours = (hours: string): boolean => {
   return validateForRange(hours, 0, 23)
 }
 
-const hasValidDays = (days: string): boolean => {
-  return validateForRange(days, 1, 31)
+const hasValidDays = (days: string, allowBlankDay?: boolean): boolean => {
+  return allowBlankDay && isQuestionMark(days) || validateForRange(days, 1, 31)
```

Comment:
Maybe add `()` around the check for clarity? Checks with `&&` and `||` mixed together aren't always obvious in their precedence, although in this case it would work (`&&` before `||` according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence)

```suggestion
  return (allowBlankDay && isQuestionMark(days)) || validateForRange(days, 1, 31)
```

Reply:
Yes, I agree, the priority will be more explicit if we add `()`